src/moduler/aktivitet/visning/avtalt-container/aktivitet/KanIkkeSendeForhaandsorienteringInfotekst.tsx

```diff
@@ -17,24 +17,22 @@ const getTekst = (
     mindreEnnSyvDagerTil: boolean
 ): string | undefined => {
     if (brukerErManuell) {
-        return 'Du kan ikke legge til forhåndsorientering fordi brukeren er manuell bruker.';
+        return 'Du kan ikke legge til forhåndsorientering fordi brukeren har manuell oppfølging. Du skal ha orientert brukeren om mulig konsekvens for ytelse, og dokumentert dette.';
     }
     if (brukerErUnderKvp) {
         return 'Du kan ikke legge til forhåndsorientering fordi brukeren deltar i kvalifiseringsprogrammet.';
     }
     if (brukerErIReservasjonKRR) {
-        return 'Du kan ikke legge til forhåndsorientering fordi brukeren har reservert seg i kontakt- og reservasjonsregisteret.';
+        return 'Du kan ikke legge til forhåndsorientering fordi brukeren har reservert seg i kontakt- og reservasjonsregisteret. Du skal ha orientert brukeren om mulig konsekvens for ytelse, og dokumentert dette.';
     }
     if (!harLoggetInnMedNivaa4) {
-        return 'Du kan ikke legge til forhåndsorientering fordi brukeren ikke har vært innlogget de siste 18 månedene med nivå 4 (for eksempel BankID).';
+        return 'Du kan ikke legge til forhåndsorientering fordi brukeren ikke har vært innlogget de siste 18 månedene med nivå 4 (for eksempel BankID). Du skal ha orientert brukeren om mulig konsekvens for ytelse, og dokumentert dette.';
     }
     if (mindreEnnSyvDagerTil) {
         return 'Du kan ikke legge til forhåndsorientering fordi sluttdatoen er færre enn 7 dager frem i tid. Hvis du har hatt muntlig dialog med brukeren om mulige konsekvenser for ytelse og dokumentert dette i et samtalereferat, så kan du sette aktiviteten til “Avtalt med NAV".';
     }
 };
 
-//TODO: slette tekstfil sett-avtalt-forhandsrientering-bruker-uten-aktivitesplan
-//TODO: Husk å slette tekstfil sett-til-avtalt-mindre-enn-syv-dager
```

Comment:
tror disse liger der for at vi skal huske og slette noen filer når vi er ferdig med fho del 1